### PR TITLE
feat: glean metrics data review

### PR DESCRIPTION
**Pull request by taddes**
## Description

Per the Glean server-side collection cross-platform telemetry setup, the configuration of Glean requires a data review. This task involves the preparation of the data review document to be reviewed by a data steward. The steward, in this case, will be Travis Long , as he has been assisting us in the process of implementing the metrics for DAU using Glean. 

Subsequent filing of a bug for review will follow and a link to said data-review bug will be within the metrics.yaml file for the application.

See relevant documentation page for data review [here](https://wiki.mozilla.org/Data_Collection)

## Testing
Requires data-steward review (in this case, Travis Long)

## Issue(s)

Closes [SYNC-4435](https://mozilla-hub.atlassian.net/browse/SYNC-4435).

[SYNC-4435]: https://mozilla-hub.atlassian.net/browse/SYNC-4435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by taddes** [comment]
Before submitting for data review, wanted to get everyone's thoughts in case something isn't captured in the data review request doc. A bug is already ready for Travis Long to review and he'll also take a look at this PR. @pjenvey @jrconlin @ddurst

File: glean/data-review/request.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  </tr>
+  <tr>
+    <td>sync_event</td>
+    <td>Event to record an instance of sync backend activity initiated by client.</td>
+    <td>Cat 2: Interaction Data</td>
+    <td><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1923967">https://bugzilla.mozilla.org/show_bug.cgi?id=1923967</a></td>
+  </tr>
+  <tr>
+    <td>hashed_fxa_uid</td>
+    <td>Sync user identifier. Uses `hashed_fxa_uid` for accurate count of sync actions.</td>
```
Comment (jrconlin) [inline glean/data-review/request.md:46]: May want to explain what the 'hashed_fxa_uid' is. (e.g. "This is the Firefox Accounts (FxA) User Identifier (UID) value passed through a SHA-256 hash to render a value that is unique, but ensures the privacy of the original UID.") It's usually best to over explain than under explain in these sorts of documents.

File: glean/data-review/request.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    <td>Cat 2: Interaction Data</td>
+    <td><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1923967">https://bugzilla.mozilla.org/show_bug.cgi?id=1923967</a></td>
+  </tr>
+  <tr>
+    <td>platform</td>
+    <td>Platform from which sync action was initiated: Desktop, Android, or iOS.</td>
```
Comment (jrconlin) [inline glean/data-review/request.md:52]: Are we doing any sort of "normalization" of these values (e.g. are we reducing Samsung, Google, and LG devices to "Android", or are we capturing the explicit value from the UserAgent string and recording that?

Comment (taddes) [reply glean/data-review/request.md:52]: We're going to normalize, yes. Won't need that granular specificity. The enum for`Platform` will reduce to Desktop, Andriod, iOS and 'DeviceFamily` will be tablet, mobile, desktop.

File: glean/data-review/request.md (hunk 1 above)
Comment (taddes) [reply glean/data-review/request.md:46]: Absolutely, I'll add this context. Ty

File: glean/data-review/request.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+* We will measure all active use of Sync (attributed from the server-side). This includes all countries, locales and release channels which Sync is used.
+
+9) If this data collection is default on, what is the opt-out mechanism for users?
+* Opting out would involve a user not being signed into Sync as a service. Since there is no active attribution that could identify a user, especially since we are using a hash value and generalized collection representation, there is not an immediate user concern.
+
+10) Please provide a general description of how you will analyze this data.
+
+* Measurement of Daily Active Use:
+- Based on the individual unique user_id (`hashed_fxa_uid`), we will count a single active user when, during a 24-hour period, the user makes a request to any collection within their Sync account (including bookmarks, tabs, prefs, etc.).
```
[the remark this answers is not in the document]
Comment (taddes) [reply glean/data-review/request.md:97]: Sounds good!

**Review by jrconlin — APPROVED** [review]

**Review by pjenvey — APPROVED** [review]

**Pull request merged**